src/setup/txt_joyaxis.c

```diff
@@ -413,6 +413,25 @@ void TXT_ConfigureJoystickAxis(txt_joystick_axis_t *joystick_axis,
     joystick_axis->callback = callback;
 }
 
+void TXT_ConfigureGamepadAxis(txt_joystick_axis_t *joystick_axis,
+                              int using_button,
+                              txt_joystick_axis_callback_t callback)
+{
+    // Build the prompt window.
+
+    joystick_axis->config_window = TXT_NewWindow("Configure axis");
+    TXT_SetColumnWidths(joystick_axis->config_window, 14);
+    TXT_AddWidgets(joystick_axis->config_window, TXT_NewStrut(0, 1),
+                   TXT_NewCheckBox("Invert", joystick_axis->invert),
```

Comment:
Does the order of `TXT_` calls matter? If so this can produce unpredictable results because parameter evaluation order is not fixed.

Reply:
No, the order of `TXT_` calls should not matter here.